### PR TITLE
[lldb] Add test for swift generic parameterized by C++ types

### PR DESCRIPTION
**Pull request by augusto2112**
rdar://100284870

### CODE REVIEW

**Comment by augusto2112** [comment]
@swift-ci test

File: lldb/test/API/lang/swift/cxx_interop/forward/swift-generic-with-cxx-type/TestSwiftGenericWithCxxType.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        self.build()
+        self.runCmd('setting set target.experimental.swift-enable-cxx-interop true')
+        _, _, _, _= lldbutil.run_to_source_breakpoint(
+            self, 'Set breakpoint here', lldb.SBFileSpec('main.swift'))
+
+        self.expect('v classWrapper', substrs=['Wrapper<CxxClass>', 't', 'a1 = 10',
+            'a2 = 20', 'a3 = 30'])
+
+        self.expect('v subclassWrapper', substrs=['Wrapper<CxxSubclass>', 't', 
+            'CxxClass = (a1 = 10, a2 = 20, a3 = 30)', 'a4 = 40'])
```
Comment (adrian-prantl) [inline lldb/test/API/lang/swift/cxx_interop/forward/swift-generic-with-cxx-type/TestSwiftGenericWithCxxType.py:24]: why don't we test expr here? Does it fail?

**Review by adrian-prantl — APPROVED** [review]

File: lldb/test/API/lang/swift/cxx_interop/forward/swift-generic-with-cxx-type/TestSwiftGenericWithCxxType.py (hunk 1 above)
Comment (augusto2112) [reply lldb/test/API/lang/swift/cxx_interop/forward/swift-generic-with-cxx-type/TestSwiftGenericWithCxxType.py:24]: No, just forgot about it :)

**Comment by augusto2112** [comment]
@swift-ci test

**Comment by augusto2112** [comment]
@swift-ci test

**Pull request merged**